lib/client-api/src/story_store.ts

```diff
@@ -401,6 +405,7 @@ export default class StoryStore {
 
       return {
         ...data,
+        parameters: this.combineStoryParameters(data.parameters, data.kind),
```

Comment:
It wasn't exactly clear which APIs we should produce denormed/combined parameters for on the story store. Right now it may be very confusing which ones we do it for.